drills/ethers-typechain-contract/drill.ts

```diff
@@ -13,109 +13,70 @@ async function drill() {
             minLevel: 3,
         },
     });
-    indexer.on("AddClaimAttestation", async (event) => {
+    indexer.on("AddClaimAttestation", async (event, { event: eventKey, hash, i, block }) => {
         console.log("Found AddClaimAttestation event");
-        await appendFile(
-            "./bridge-events.txt",
-            `${event.event} ${event.transactionHash} ${event.logIndex} ${event.blockNumber} ${JSON.stringify(event)}\n`,
-        );
-        indexer.eventDone("AddClaimAttestation", event.transactionHash, event.logIndex);
+        await appendFile("./bridge-events.txt", `${eventKey} ${hash} ${i} ${block} ${JSON.stringify(event)}\n`);
```

Comment:
Una mica out of context xo si q el codi es repeteix tota la estona no? Podries fer una funció.. 
Només canvia el primer parametre del `indexer.eventDone("name", ...)`.